File: Resolvendo exercicíos em Python/secao07/Exercicio06.py
num = int(input("Informe um número entre 1 e 10: "))

while num < 1 or num > 10:
    num = int(input("Informe um número entre 1 e 10: "))
print(f"Tabuada do {num}")
for i in range(1, 11):
    print(f"{num} x {i} = {num * i}")

File: Resolvendo exercicios em JAVA/src/secao07/Exercicio06.java
package secao07;

import java.util.Scanner;

public class Exercicio06 {

	public static void main(String[] args) {
		int numero;
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Informe um numero: ");
		numero = teclado.nextInt();
		
		while(numero < 1 || numero > 10) {
			System.out.println("Informe um número entre 1 e 10: ");
			numero = teclado.nextInt();
		}
		for(int i = 1; i <= 10; i++ ) {
			System.out.printf("%d X %d = %d\n", numero, i, (numero * i));
		}
		
		teclado.close();
	}

}
